_neutron/HAProxy_Configuration.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: page
+title:  "HAProxy Configuration for Large Number of Networks"
+featured: true
+weight: 2
+tags: [neutron, haproxy, configuration]
+author: Peter Xiang Wang, Sangeetha Srikanth
+dateAdded: June 1st, 2016
+editor: Leslie Lundquist
+---
+
+## HAProxy Configuration for Large Number of Networks
+
+When you connect to a large number of Neutron networks, the default HAProxy buffer size will not be large enough for the Neutron CLI to list such a large number. You will see the following error message:
```

Comment:
@ssrikant20 Sangeetha, is this document intended for customers or operations staff?